project/src/main/puzzle/piece/piece-physics.gd

```diff
@@ -58,7 +58,7 @@ func move_piece(piece: ActivePiece) -> bool:
 		result = true
 		if piece.lock > 0:
 			if dropper.did_hard_drop:
-				# hard drop doesn't cause lock reset
+				# hard drop doesn't cause lock cancel
```

Comment:
Revert this; they're referring to a lock reset which is different from a lock cancel

Reply:
Reverted